1073544 - net-snmp.rpm is not multilib safe
Based on this upstream commit:

commit f4d301f7b38b9f56397a4a4ed90f7e21cdcc5753
Author: Jan Safranek <jsafranek@users.sourceforge.net>
Date:   Thu Jan 22 13:24:12 2015 +0100

    Removed unused variables from net-snmp-create-v3-user.


diff -up net-snmp-5.5/net-snmp-create-v3-user.in.multilib net-snmp-5.5/net-snmp-create-v3-user.in
--- net-snmp-5.5/net-snmp-create-v3-user.in.multilib	2015-01-22 13:18:04.438642840 +0100
+++ net-snmp-5.5/net-snmp-create-v3-user.in	2015-01-22 13:17:25.000000000 +0100
@@ -5,29 +5,6 @@
 # this shell script is designed to add new SNMPv3 users
 # to Net-SNMP config file.
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-includedir=@includedir@
-libdir=@libdir@
-datarootdir=@datarootdir@
-NSC_LDFLAGS="@LDFLAGS@"
-NSC_INCLUDEDIR=${includedir}
-NSC_LIBDIR=-L${libdir}
-NSC_LIBS="@LIBS@"
-NSC_AGENTLIBS="@AGENTLIBS@ @PERLLDOPTS_FOR_APPS@"
-NSC_PREFIX=$prefix
-NSC_EXEC_PREFIX=$exec_prefix
-NSC_SRCDIR=@srcdir@
-NSC_INCDIR=${NSC_PREFIX}/include
-NSC_BASE_SUBAGENT_LIBS="-lnetsnmpagent -lnetsnmphelpers -lnetsnmp"
-NSC_BASE_AGENT_LIBS="-lnetsnmpagent -lnetsnmphelpers -lnetsnmpmibs -lnetsnmp"
-NSC_SRC_LIBDIRS="agent/.libs snmplib/.libs agent/helpers/.libs"
-NSC_SRC_LIBDEPS="agent/.libs/libnetsnmpmibs.a agent/.libs/libnetsnmpagent.a agent/helpers/.libs/libnetsnmphelpers.a snmplib/.libs/libnetsnmp.a"
-
-if test "x$NSC_SRCDIR" = "x." ; then
-   NSC_SRCDIR="NET-SNMP-SOURCE-DIR"
-fi
-
 if @PSCMD@ | egrep ' snmpd *$' > /dev/null 2>&1 ; then
     echo "Apparently at least one snmpd demon is already running."
     echo "You must stop them in order to use this command."
